spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
spring.datasource.url=jdbc:mysql://localhost:3306/allivery?useSSL=false&allowPublicKeyRetrieval=true&useUnicode=true&serverTimezone=Asia/Seoul
spring.datasource.username=zero
spring.datasource.password=connect123
spring.jpa.show-sql=true
spring.jpa.properties.hibernate.format_sql=true

spring.jpa.hibernate.ddl-auto=update
spring.mvc.pathmatch.matching-strategy=ant_path_matcher

spring.messages.basename=i18n/exception
spring.messages.encoding=UTF-8

spring.jwt.secret=pepega524832857240sdaf

unKnown.code=-9999
unKnown.msg=An unknown error has occurred. SadPepe:(
userNotFound.code=-1000
userNotFound.msg=This member not exist. SadPepe:(
emailSigninFailed.code=-1001
emailSigninFailed.msg=Your account does not exist or your email or password is incorrect. SadPepe :(

spring.cloud.gateway.routes={id=auth, uri=lb://AUTH-SERVICE, predicates=[Path=/auth-service/check/**, Method=GET, Path=/auth-service/logout, Method=POST], filters=[RemoveRequestHeader=Cookie, RewritePath=/auth-service/(?<segment>.*), /$\{segment}]}

logging.level.org.springframework=debug
logging.level.org.springframework.web=debug

spring.devtools.livereload.enabled=true
spring.devtools.restart.enabled=false
spring.freemarker.cache=false
spring.jackson.serialization.fail-on-empty-beans=false

